client/src/api/applicants.js

```diff
@@ -1,8 +1,14 @@
-const getAllApplicants = () => {
+export const getAllApplicants = () => {
   return fetch("/api/applicantProfile")
     .then(res => res.json())
     .catch(err => {
       console.error(err);
     });
 };
-export default getAllApplicants;
+export const getSkillsByApplicantId = id => {
+  return fetch(`/api/applicantSkills/${id}`)
```

Comment:
can you makes the endpoint `api/applicants/${id}/skills` .. this is more meaningful and follows REST closer as the ID is for the Applicant, not the skills.

Read more about REST here .. https://stackoverflow.com/questions/671118/what-exactly-is-restful-programming

Reply:
done